### ISSUE TITLE
Poor Performances on Test Set

### ISSUE THREAD

**Issue by GerardoFesta** [GerardoFesta/3DFER_SE4AI#1]
Both models are likely overfitting (train accuracy is way higher than test accuracy). Need for a strategy to avoid that.

1. Normalization of the inputs
2. Plot of accuracy on train and test during the training epochs
3. Assess whether there's a cutoff point where overfit starts.

If the steps above aren't successful, we should try to use a preprocessing pipeline.
